source/lib/runbook_factory.ts

```diff
@@ -40,19 +40,9 @@ export class RunbookFactory extends Construct {
   }
 
   static createControlRunbook(scope: Construct, id: string, props: IssmPlaybookProps): CfnDocument {
-    let scriptPath = '';
-    if (props.scriptPath == undefined) {
-      scriptPath = `${props.ssmDocPath}/scripts`;
-    } else {
-      scriptPath = props.scriptPath;
-    }
+    const scriptPath = props.scriptPath ? props.scriptPath : `${props.ssmDocPath}/scripts`;
```

Comment:
Replaced these two ifs with a ternary operator to lower code complexity.

Reply:
I often get the feeling that ternary operators are less clear than a simple if-else more often than not. In this case I think sonarqube is probably right that it's easier to read.

Does anybody else find ternaries harder to read than say:
```
if(condition) result
else otherResult
```